molecule/test/functional/conftest.py

```diff
@@ -277,6 +282,30 @@ def supports_docker():
     return True
 
 
+@pytest.helpers.register
+@util.lru_cache()
+def supports_podman():
+    # Returns true if podman is supported and working
+    # Returns false if podman in not supported
+    # Calls pytest.fail if podman appears to be broken
+    if min_ansible("2.8.6") and platform.system() != 'Darwin':
+        podman = get_podman_executable()
+        if podman:
+            result = util.run([podman, "info"], stdout=PIPE, universal_newlines=True)
```

Comment:
I'd rather use "podman ps", it's really checking if podman is functioning on the host